shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/api/impl/GovernanceRepositoryAPIImplTest.java

```diff
@@ -65,16 +67,19 @@ public final class GovernanceRepositoryAPIImplTest {
     
     private static GovernanceRepositoryAPI governanceRepositoryAPI;
     
+    private static RuleAlteredJobAPI ruleAlteredJobAPI;
+    
```

Comment:
It's better not depend on RuleAlteredJobAPI, since RuleAlteredJobAPIImpl depend on GovernanceRepositoryAPI